model/project.go

```diff
@@ -828,6 +828,18 @@ func PopulateExpansions(t *task.Task, h *host.Host, oauthToken string) (util.Exp
 		expansions.Put("revision_order_id", fmt.Sprintf("%s_%d", v.Author, v.RevisionOrderNumber))
 		if v.Requester == evergreen.MergeTestRequester {
 			expansions.Put("is_commit_queue", "true")
+			patch, err := patch.FindOne(patch.ByVersion(t.Version))
```

Comment:
Seems shady to use the library name as the variable

Reply:
+1

Reply:
Fixed.